odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewStats/OverviewDatasetStats/OverviewDatasetStats.tsx

```diff
@@ -18,53 +18,46 @@ interface OverviewDatasetStatsProps extends StylesType {
 }
 
 const OverviewDatasetStats: React.FC<OverviewDatasetStatsProps> = ({
-  classes,
   stats,
 }) => (
   <Grid container>
-    <Grid item xs={12} className={classes.typeLabel}>
+    <Grid item xs={12} sx={{ ml: 0, mb: 1.25 }}>
       <EntityTypeItem typeName={DataEntityTypeNameEnum.SET} fullName />
     </Grid>
-    <Grid item container xs={4} className={classes.statsItem}>
+    <StatsItem item container xs={4}>
       <Grid item xs={12}>
-        <Typography variant="h6" className={classes.statLabel}>
-          Rows
-        </Typography>
+        <StatLabel variant="h6">Rows</StatLabel>
       </Grid>
       <Grid item container xs={12} wrap="nowrap" alignItems="center">
-        <RowsIcon className={classes.statIcon} />
-        <Typography variant="h2" className={classes.statValue}>
+        <RowsIcon sx={{ fontSize: 'h5.fontSize', mr: 1 }} />
+        <Typography variant="h2">
           <NumberFormatted value={stats?.rowsCount} />
         </Typography>
       </Grid>
-    </Grid>
-    <Grid item container xs={4} className={classes.statsItem}>
+    </StatsItem>
+    <StatsItem item container xs={4}>
       <Grid item xs={12}>
-        <Typography variant="h6" className={classes.statLabel}>
-          Columns
-        </Typography>
+        <StatLabel variant="h6">Columns</StatLabel>
       </Grid>
       <Grid item container xs={12} wrap="nowrap" alignItems="center">
-        <ColumnsIcon className={classes.statIcon} />
-        <Typography variant="h2" className={classes.statValue}>
+        <ColumnsIcon sx={{ fontSize: 'h5.fontSize', mr: 1 }} />
+        <Typography variant="h2">
           <NumberFormatted value={stats?.fieldsCount} />
         </Typography>
       </Grid>
-    </Grid>
-    <Grid item container xs={4} className={classes.statsItem}>
+    </StatsItem>
+    <StatsItem item container xs={4}>
       <Grid item xs={12}>
-        <Typography variant="h6" className={classes.statLabel}>
-          Used by
-        </Typography>
+        <StatLabel variant="h6">Used by</StatLabel>
       </Grid>
       <Grid item container xs={12} wrap="nowrap" alignItems="center">
-        <UserIcon className={classes.statIcon} />
-        <Typography variant="h2" className={classes.statValue}>
+        <UserIcon sx={{ fontSize: 'h5.fontSize', mr: 1 }} />
```

Comment:
Please in sx prop use only margin or padding properties.
Sizes of icons (RowsIcon, UserIcon) doesn't match with sizes from design. You should compare these three icons components code and refactor two of these. Then you can use right styles without fontSize prop